[2023-09-22T16:52:34.768+0000] {logging_mixin.py:149} WARNING - /home/***/.local/lib/python3.9/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-09-22T16:52:35.024+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v02.greet__1 scheduled__2021-11-13T00:00:00+00:00 [queued]>
[2023-09-22T16:52:35.034+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v02.greet__1 scheduled__2021-11-13T00:00:00+00:00 [queued]>
[2023-09-22T16:52:35.036+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 6
[2023-09-22T16:52:35.097+0000] {taskinstance.py:1350} INFO - Executing <Task(_PythonDecoratedOperator): greet__1> on 2021-11-13 00:00:00+00:00
[2023-09-22T16:52:35.118+0000] {standard_task_runner.py:57} INFO - Started process 591 to run task
[2023-09-22T16:52:35.149+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_taskflow_api_v02', 'greet__1', 'scheduled__2021-11-13T00:00:00+00:00', '--job-id', '151', '--raw', '--subdir', 'DAGS_FOLDER/test2.py', '--cfg-path', '/tmp/tmpc8nmy0b4']
[2023-09-22T16:52:35.157+0000] {standard_task_runner.py:85} INFO - Job 151: Subtask greet__1
[2023-09-22T16:52:35.209+0000] {logging_mixin.py:149} WARNING - /home/***/.local/lib/python3.9/site-packages/***/settings.py:188 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-09-22T16:52:35.322+0000] {task_command.py:410} INFO - Running <TaskInstance: dag_with_taskflow_api_v02.greet__1 scheduled__2021-11-13T00:00:00+00:00 [running]> on host 75cee4990e54
[2023-09-22T16:52:35.393+0000] {logging_mixin.py:149} WARNING - /home/***/.local/lib/python3.9/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-09-22T16:52:36.904+0000] {taskinstance.py:1568} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_taskflow_api_v02' AIRFLOW_CTX_TASK_ID='greet__1' AIRFLOW_CTX_EXECUTION_DATE='2021-11-13T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-11-13T00:00:00+00:00'
[2023-09-22T16:52:36.912+0000] {logging_mixin.py:149} INFO - Hello World! My name is Jerry Fridman and I am 19 years old!
[2023-09-22T16:52:36.914+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-09-22T16:52:36.933+0000] {taskinstance.py:1368} INFO - Marking task as SUCCESS. dag_id=dag_with_taskflow_api_v02, task_id=greet__1, execution_date=20211113T000000, start_date=20230922T165235, end_date=20230922T165236
[2023-09-22T16:52:37.041+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-09-22T16:52:37.916+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
